docs/src/process/release.md

```diff
@@ -3,25 +3,57 @@
+
+Major releases are necessitated by fundamental framework changes that are expected to be incompatible
+with previous releases.
+
+Minor releases are necessitated by non-backwards-compatible application, API changes, or new
+features or enhancements.
+
+Patch releases are created for backporting fixes to blocking bugs that were discovered _after_
+the release of a major or minor version. They are not to introduce new features, enhancements, or
+dependency changes.
 
 ## 3. Changelog Maintenance
 
 A comprehensive changelog file, `CHANGELOG.md`, documents any changes, adding a high level of transparencies for anyone desiring to look into the status of new and past progress. It includes the summation of any major new enhancements, changes, bug fixes, and the credits to the users responsible for each unique progress.
 
 ## 4. Notable Changes Labels on GitHub PRs
 
-For the Open MCT package, we leverage GitHub's Pull Request (PR) mechanisms extensively, with three important PR labels dedicated to signifying 'notable_changes':
+Pull requests are to be labeled with "notable_change" if they contain changes that fit any of the 
+following criteria:
 
-- **Breaking Change** Highlights the integration of changes that are suspected to break, or without a doubt will break, backward compatibility. These should signal to users the upgrade might be seamless only if dependency and integration factors are properly managed, if not, one should expect to manage atypical technical snags.
-- **API change** Signifies when a contribution makes any complete or under layer changes to the communication or its supporting access processes. This label flags required see-through insight on how the web-based control panel sees and manipulates any value and or network logs.
-- **Default Behavior Change:** In the incident an update either adjusts a form to or integrates a not previously kept setting or plugin. i.e. autoscale is enabled by default when working with plots.
+- **Breaking Change**
+    - Highlights the integration of changes that are suspected to break, or without a doubt will break, backwards compatibility. These should signal to users the upgrade might be seamless only if dependency and integration factors are properly managed, if not, one should expect to manage atypical technical snags.
+- **API Change**
+    - Signifies when a contribution makes any complete or under layer changes to the communication or its supporting access processes. This label flags required see-through insight on how the web-based control panel sees and manipulates any value and or network logs.
+- **Default Behavior Change**
+    - In the incident an update either adjusts a form to or integrates a not previously kept setting or plugin. i.e. autoscale is enabled by default when working with plots.
 
 ## 6. Community & Contributions
 
```

Comment:
Add section on deprecation.

Summary Widget Deprecated Example. 
- Notify Users
- Supposed to write interceptor to convert to conditionWidgets
- Deprecation warning